internal/blocktx/blockchain_communication/p2p/init.go

```diff
@@ -1,25 +1,22 @@
-package blocktx
+package blocktx_p2p
 
 import (
 	"encoding/binary"
-	"errors"
 	"io"
-	"log/slog"
+
+	blockchain "github.com/bitcoin-sv/arc/internal/blocktx/blockchain_communication"
```

Comment:
`blockchain_communication` is a very long package name
How about `peer_communication`?

Reply:
Yeah, I know... I don't like it either. But I have in mind that we'll add multicast soon, and I think the code will eventually be arranged something like this:
```
blocktx
   |
   |- blockchain_comunication
           |
           |- p2p/
           |- multicast/
           | some files to contoll how to use p2p and multicast
           
So, `blockchain_communication` isn't ideal (maybe it's bad), but `peer_communication` doesn't accurately describe what code will ultimately include.

Reply:
Couple of ideas for better name for the package:
- `blockchain`
- `network`
- `blocknet`
- `blockchain_network`
- `chain_communication`
- `netlayer`
- `bcnet`
- `nodes_communication`

Reply:
I'll change this name in follow up PR